engine/schema/src/com/cloud/vm/dao/UserVmDaoImpl.java

```diff
@@ -645,8 +643,9 @@ public boolean remove(Long id) {
 
         PreparedStatement pstmt = null;
         try {
-            pstmt = txn.prepareStatement(VMS_DETAIL_BY_NAME + detail + VMS_DETAIL_BY_NAME2 + getQueryBatchAppender(vmNames.size()));
-            int i = 1;
+            pstmt = txn.prepareStatement(VMS_DETAIL_BY_NAME + getQueryBatchAppender(vmNames.size()));
```

Comment:
Yes, it fixes it :)
Did a local build to check.. findbugs is smart enough to check that getQueryBatchAppender only adds static " ?,"  string